src/modules/delete.js

```diff
@@ -0,0 +1,17 @@
+const listItems = document.getElementById('list-container');
+const saveData = () => {
+  localStorage.setItem('data', listItems.innerHTML);
+};
+
+export default function clearCheckedItems() {
+  const listItems = document.getElementById('list-container').children;
+
+  for (let i = 0; i < listItems.length; i += 1) {
+    if (listItems[i].classList.contains('checked')) {
+      listItems[i].remove();
+      i -= 1;
+    }
+  }
+
+  saveData();
+}
```

Comment:
- [ ] Kindly ensure that when a task is removed, the `index` properties of the remaining items are updated per the requirement.

  ![image](https://github.com/AleWaweru/TODO-LIST/assets/54848363/0002b1e6-49d5-450e-9fd9-1ce74454d30c)

Reply:
done.